docs/beamrunner/release/1.2.0/io.md

```diff
@@ -14,89 +14,75 @@ next:
-
-1. After the service is provisioned, select the object storage service that you created from the dashboard to open the **Manage** page for the service.
-2. On the service page, click **Service credentials**.
-3. If necessary, create a credential by clicking **New credential**. Use the default information and click **Add**.
-4. Click **View credentials**.
-5. On the computer where Streams Runner is installed, create the following environment variables from the fields that are shown in the credentials.  
-
-| Environment variable | Command-line option           | Credentials field | Environment variable example                                                 |
-|----------------------|------------------|-------------------|---------------------------------------------------------|
-| `OS_USER_ID`           | `--swiftUserId`    | `userId`            | `export OS_USER_ID='2b670d77432e4cf2bd128ef9ff61fa56'`    |
-| `OS_PASSWORD`          | `--swiftPassword`  | `password`          | `export OS_PASSWORD='f1H/~BIO.=s0wuT9'`                  |
-| `OS_PROJECT_ID`        | `--swiftProjectId` | `projectId`         | `export OS_PROJECT_ID='80301e24254f4ffb81d53f0cddccad78'` |
-| `OS_REGION_NAME`       | `--swiftRegion`    | `region`            | `export OS_REGION='dallas'`                               |
+To use the storage service from Beam applications, you must specify the IBM Cloud service credentials. Since the `s3` FileSystem is based on AWS object storage, you will need to create credentials with HMAC keys.
 
-**Tip**: For MacOS, the Swift command of OpenStack might collide with the existing Xcode Swift command. To avoid the conflicts, create a Python virtual environment, and install the Swift client in the virtual environment.
+1. On the service page, click **Service credentials**.
+1. Create a credential by clicking **New credential**.
+1. Put `{"HMAC":true}` in the **Add Inline Configuration Parameters (Optional) field** and click **Add** to create the new credentials.
+1. Click **View credentials** on the newly added credentials.
+1. On the computer where Streams Runner is installed, create the following environment variables from the fields that are shown in the credentials.
 
-```bash
-virtualenv my_project
-cd my_project
-source bin/activate
-pip install python-swiftclient
-pip install python-keystoneclient
-```
+| Environment variable | Credentials field | Environment variable example|
+|----------------------|-------------------|-----------------------------|
+| `AWS_ACCESS_KEY_ID` | `cos_hmac_keys.access_key_id` | `export AWS_ACCESS_KEY_ID=de4e2e3d7bd943a99b672f13dec40f7c`|
+| `AWS_SECRET_ACCESS_KEY` | `cos_hmac_keys.secret_access_key` | `export AWS_SECRET_ACCESS_KEY=54f077c504ebef49bf707cc0d57e3f2a4f4d4a6898b53fec`|

```

Comment:
Should we describe how to get these values to the app with the `--awsCredentialsProvider` option?